### PR TITLE
Remove logging to emby servers

### LINKED ISSUE #6: Remove log sending / nullify url
See: https://github.com/jellyfin/jellyfin-androidtv/blob/debef952817992e385068cea31bd07dceb392a30/MediaBrowserTv/app/src/main/java/tv/emby/embyatv/util/LogReporter.java#L54

### PR DESCRIPTION
**Pull request by bfayers**
Resolved #6

### CODE REVIEW

File: MediaBrowserTv/app/src/main/java/tv/emby/embyatv/util/Utils.java [hunk 1]
```diff
@@ -1282,7 +1282,7 @@ public void onResponse(ConnectionResult serverResult) {
 
             @Override
             public void onError(Exception exception) {
-                reportError(activity, activity.getString(R.string.msg_error_connecting_server));
+                showToast(context, context.getString(R.string.msg_report_sent));
```
Comment (dkanada) [inline MediaBrowserTv/app/src/main/java/tv/emby/embyatv/util/Utils.java]: Why did you change this section?

Comment (bfayers) [reply MediaBrowserTv/app/src/main/java/tv/emby/embyatv/util/Utils.java]: Good question. Will revert

Comment (bfayers) [reply MediaBrowserTv/app/src/main/java/tv/emby/embyatv/util/Utils.java]: @dkanada 159f3bc

**Comment by dkanada** [comment]
I would drop the original commit rather than make a new commit that only reverts a previous commit in a pull request =p

**Comment by bfayers** [comment]
Branch became cursed. Trying again with a new PR
